tests/Database/Consensus/Elements/TestElements.nim

```diff
@@ -133,7 +133,7 @@ proc newRandomMeritRemoval*(
     holder: uint16 = uint16(rand(high(int16)))
 ): SignedMeritRemoval =
     var
-        partial: bool = rand(1) == 1
+        partial: bool = rand(1) == 0
```

Comment:
Was there a reason for this? I don't mind it if you think it's clearer, just curious.

Reply:
Simply for consistency. This was the only place we used 1.